client/src/views/workspaces/WorkspacesPage.vue

```diff
@@ -84,7 +84,9 @@
         </div>
       </div>
       <div class="workspaces-footer title-h5">
-        {{ $t('WorkspacesPage.itemCount', { count: workspaceList.length }, workspaceList.length) }}
+        <div class="folder-footer__container">
```

Comment:
Maybe not your fault, but it seems that the counter on the workspace page is black, whereas the counter on other pages (users or files) is gray.